mars/resource.py

```diff
@@ -391,3 +391,25 @@ def cuda_card_stats() -> List[_cuda_card_stat]:  # pragma: no cover
             )
         )
     return infos
+
+
+def build_band_resources(
```

Comment:
`band` is a concept introduced in Cluster Service. Thus not appropriate to introduce it in `mars.resource`.

Reply:
So, where is the best place for the `build_band_resources`?

Reply:
Fixed. Thanks.